make/utils.cmake

```diff
@@ -221,4 +221,40 @@ if (UNIX)
     append_property_string(TARGET ${target} LINK_FLAGS "${ldflags}")
   endfunction (set_preferred_base_start_and_end)
 
+  function (check_avx_processor_and_compiler_support out)
```

Comment:
I think this should not be inside the "if (UNIX)" for the above function, because these are not inherently UNIX-only.  Maybe it would be better to be top-level-indent (to avoid having to reindent the whole thing when Windows is added, which should be pretty easy to do: 2-line change) with an "assert" == `if (WIN32) message(FATAL_ERROR` or sthg.

Reply:
Moved outside of UNIX and added error message instead.